frontend/src/components/dashboard/hooks/useDissimilarAnalyses/index.ts

```diff
@@ -48,17 +50,25 @@ const computePairwiseDistance = async (
   return distances;
 };
 
+/**
+ * Computes the best combination of analyses.
+ * @param numberOfAnalyses The number of analyses to select.
+ * @param distances The distance matrix of the analyses to choose from
+ * @param initialBestDistance The initial distance to start from for the best combination.
+ * @param initialCombinationDistance The initial distance to start from for each combination.
+ * @param combineDistances A function that aggregates the distances of the analysis within a combination.
+ * @param isNewDistanceBetter A function that checks if a newly aggregated combination distance is better than the best distance.
+ * @returns The indices  of the analysis into the distance matrixthat are the best combination.
+ */
 const getBestCombination = (
   numberOfAnalyses: number,
   distances: number[][],
-  initialDistance: number = 0,
-  combineDistances: (overallDistance: number, distance: number) => number = (
-    d1,
-    d2,
-  ) => d1 + d2,
+  initialBestDistance: number,
+  initialCombinationDistance: number,
+  combineDistances: (overallDistance: number, distance: number) => number,
```

Comment:
Don't change this now, but in the future we might want to take an array here - so that we can implement things like quadratic mean.